certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/model/Certificate.java

```diff
@@ -236,6 +236,7 @@ private void throwIfConcurrentModification(Revision revision, String operation,
   }
 
   public void send(ActionEvaluation actionEvaluation) {
+    final var optionalActionEvaluation = Optional.ofNullable(actionEvaluation);
```

Comment:
Thinking that it would make sense to add a new send method instead of using the current one and passing "null" for actionEvaluation.

Could be called send(), but could be hard to understand when to use which... so maybe something like sendByCitizen() or similar.